docs/RELEASE_CHECKLIST.md

```diff
@@ -1,5 +1,23 @@
 # Release checklist
 
+## After hard freeze
+
+- [ ] Bump all CI requirements to current versions:
+
+      ```bash
+      rm requirements-ci-current.txt requirements-ci-fenics.txt conda-lock.yml
+      make ci_requirements
+      ```
+
+      Commit all changes.
+      Before proceeding, wait until changes are merged.
+- [ ] Check if all tutorials are rendered correctly.
+- [ ] Check if binder links are functional.
+- [ ] Use `hatch build` to create wheel. `pip install` wheel (including extras) into new venv.
+      Check if installation is working as expected.
```

Comment:
How to install `hatch`? Which version to use? I don't see if it's defined.

Reply:
I have added a brief comment. As with `pip` and other packaging tools, you probably want an up-to-date versions. Usually, it should not matter, however.